lib/shanty/plugins/rspec_plugin.rb

```diff
@@ -10,6 +10,7 @@ class RspecPlugin < Plugin
       # being used from the presence of a spec directory alone (this can be many other testing frameworks!)
       provides_projects :rspec_projects
       subscribe :test, :rspec
+      description 'Discovers and runs RSpec tests on Ruby projects'
```

Comment:
Description here isn't consistent with one from the Cucumber. Would say the Cucumber one should change to `Executes` to `Discovers and runs` as well.